apps/chat/bots.py

```diff
@@ -208,6 +213,9 @@ def main_bot_chain(user_input):
             }
         return main_bot_chain.invoke(user_input, config=config)
 
+    def get_ai_message_id(self) -> int | None:
```

Comment:
This method essentialy relays information from the runnable's state to the caller and can easily be extended in the future to include other information that we might want to relay. I chose this "on-demand" method rather than returning some object as part of the original response for two reasons:
1. This way is much less effort
2. We don't have to return information when it's not going to be used. Currently only the webchannel cares about the message_id, so it can just ask for it whenever it needs it. I realize this isn't a very strong case for doing it this way in itself, but I'm going with it.